compose-app/build.gradle.kts

```diff
@@ -0,0 +1,86 @@
+plugins {
+  id("com.android.application")
+  kotlin("android")
+  kotlin("android.extensions")
+  id("com.mapbox.maps.token")
+  id("io.gitlab.arturbosch.detekt").version(Versions.detekt)
+}
+
+val buildFromSource: String by project
+
+android {
+  compileSdk = AndroidVersions.compileSdkVersion_AndroidAuto
+  defaultConfig {
+    applicationId = "com.mapbox.maps.testapp.compose"
+    minSdk = AndroidVersions.minSdkVersion_AndroidAuto
```

Comment:
`auto`?

Reply:
It's basically the same min SDK and max SDK version, so I left it to use AndroidAuto versions here.

Reply:
fixed